frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/tags/DisplayEnumInfo.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Siemens AG, 2017. Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.portal.tags;
+
+import org.apache.thrift.TEnum;
+import org.eclipse.sw360.datahandler.common.ThriftEnumUtils;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.JspWriter;
+import javax.servlet.jsp.tagext.SimpleTagSupport;
+import java.io.IOException;
+import java.util.Iterator;
+
+import static com.google.common.base.Predicates.instanceOf;
+import static com.google.common.collect.Iterables.all;
+
+/**
+ * @author thomas.maier@evosoft.com
+ */
+public class DisplayEnumInfo extends SimpleTagSupport {
+
+    private static final String SW360_TOOLTIP = "sw360-tt";
```

Comment:
same constant as in DisplayEnum.class. Also, when you pull it out, it's probably better to rename it to more descriptive `SW360_TOOLTIP_CSS_CLASS` or something like that

Reply:
Moved to Portal Constants and renamed it to TOOLTIP_CLASS__CSS